### ISSUE TITLE
VoiceOver in Chrome is not announcing lift instructions

### ISSUE THREAD

**Issue by ccveer** [atlassian/react-beautiful-dnd#1695]
### Expected behavior
VoiceOver announces the default lift instructions: "Draggable item. Ensure your screen reader is not in browse mode and then press space bar to lift.".

### Actual behavior
VoiceOver does not announce the lift instructions

### Steps to reproduce
1. Open https://react-beautiful-dnd.netlify.com/?path=/story/accessibility--single-list in Chrome
2. Turn on VoiceOver
3. Tab to one of the draggable items

### Suggested solution?
Add a default role to the draggable items that is an inter-actable role.

The reason I would fix it this way is due to the following:
![image](https://user-images.githubusercontent.com/49158600/71745410-a7191180-2e2f-11ea-9f53-08a090c92035.png)
which comes from https://www.w3.org/TR/using-aria/#label-support

I would probably use the "button" role, but I believe any of the Widget roles will work. https://www.w3.org/TR/wai-aria/#widget_roles

### What version of `React` are you using?
I see this in my implementation on version 16.12.0, but it is also occurring on whatever version is being used by your implementation I referenced in Steps to reproduce.

### What version of `react-beautiful-dnd` are you running?
I see this in my implementation on version 12.2.0, but it is also occurring on whatever version is being used by your implementation I referenced in Steps to reproduce.

### What browser are you using?
Chrome Version 79.0.3945.88 (Official Build) (64-bit)

**Comment by alexreardon** [comment]
Try opening the example out of an iframe. There is a little pop out button

**Comment by ccveer** [comment]
![image](https://user-images.githubusercontent.com/49158600/71828722-85569f00-3068-11ea-8c88-a0cd1b1b3391.png)

I clicked this button and retested.  I got the same results.

**Comment by alexreardon** [comment]
What version of your operating system are you using?

**Comment by alexreardon** [comment]
I can confirm that lift instructions are not being fired on all elements (eg div). I think we will need to look at giving the element a role if it doesn't have one already. Damn!

**Comment by alexreardon** [comment]
Ideally we do not put a role on every element as that would override the default behaviour for particular elements such as anchor. But we won't know what the tag of the drag handle elements are ahead of time. I'm not sure what to do with this.

**Comment by ccveer** [comment]
Could you surface the ability to add a role to the elements?

**Comment by alexreardon** [comment]
You can add a role today if you want to (which is an okay work around)

**Comment by alexreardon** [comment]
Ideally, we add a role to non-interactive elements by default..

**Comment by alexreardon** [comment]
I cannot see a way of conditionally adding a `role`. I think we need to just add a `role` (eg `button`) and call that out in the docs, and show how people can remove it or change it where appropriate

I think we need to push being accessible out of the box

**Comment by alexreardon** [comment]
I want to have more of a read of the spec + play with screen readers

**Comment by alexreardon** [comment]
Proposed changes:

1. role: add `role="button"` for drag handles
2. role description: add `aria-roledescription="Draggable item"` for drag handle (it looks like aria-roledescription does not work with role=button
3. lift announcement: add `aria-label` rather than `aria-labelledby` on drag handle. This will be a breaking change. I think it is worth it so that all the screen reader lift customisation can be done in the one place

**Comment by alexreardon** [comment]
looks like we cannot use `aria-roledescription` with `role=button`

**Comment by alexreardon** [comment]
Proposal:

- name: `aria-labelledby` pointing to visually hidden element with default text 'Draggable item'. Or `Draggable item - ${screenReaderFriendlyName}`.
- role: `button` (can be changed or removed by consumers)
- description: `aria-describedby` pointing to a visually hidden element with default text 'Press space bar to lift. Ensure your screen reader is not in browse mode'

This seems to be aligned to my understanding of aria and works well with lighthouse / axe

**Comment by alexreardon** [comment]
At one point I had voiceover reading the description, but now i don't. I am continuing to experiment

**Comment by alexreardon** [comment]
Any input would be super appreciated. I have what I think is the correct solution: https://github.com/atlassian/react-beautiful-dnd/blob/accessibility-refresh/docs/guides/screen-reader.md#on-focus-messaging

**Comment by alexreardon** [comment]
This will be a breaking change as we are moving from `aria-labelledby` to `aria-describedby`

**Comment by alexreardon** [comment]
@ccveer can you try on this dev build? https://deploy-preview-1741--react-beautiful-dnd.netlify.com/?path=/story/single-vertical-list--basic

**Comment by ccveer** [comment]
Everything is working as expected on my end.  Thank you!

**Comment by alexreardon** [comment]
Hi @ccveer can you please try out this one?

https://deploy-preview-1741--react-beautiful-dnd.netlify.com/?path=/story/single-vertical-list--basic

I've tested it on JAWS and NDVA, but worth getting a second look

**Comment by alexreardon** [comment]
I haven't figured out how to get spacebar to work correctly for JAWS. It seems to work correctly in 'forms mode', but i cannot figure out how to force JAWS into forms mode

**Comment by ccveer** [comment]
Just tested on VoiceOver and it works great!  Thanks so much!

**Comment by ccveer** [comment]
I've only ever used VoiceOver, Narrator, and NVDA, so I unfortunately don't know how to help you with JAWS.

**Comment by alexreardon** [comment]
That's all good. Did the new solution work with those ones @ccveer ?
